### ISSUE TITLE
[Feature Request] Support shoppify emails

### ISSUE THREAD

**Issue by jaredcat** [moralmunky/Home-Assistant-Mail-And-Packages#1031]
Add support for shipping emails from Shopify.
They seem to look like they come from: `shop+<id>@t.shopifyemail.com`

Subject lines:
`Your <item> Order <order number> is Confirmed! 🎉`
`Order <order number> confirmed`
`A shipment from order <order number> is on the way`
`Shipping update for order <order number>`

Note that Shoppify has a PoS and I have an email for a purchase made in-person with the subject
`Receipt for order <order number>`

Let me know if theres any more information needed

**Comment by firstof9** [comment]
Shoppify isn't a shipper are they?
Do they not use UPS/USPS/FedEx/etc ?

**Comment by pr8x** [comment]
@firstof9 I noticed that for some websites (like Shopify, ebay, etc) I am not receiving a dedicated DHL/DPD/whatever mail, only a custom mail from the seller. Is there any way to deal with that?

**Comment by jaredcat** [comment]
> Shoppify isn't a shipper are they? Do they not use UPS/USPS/FedEx/etc ?

Neither is a seller on Amazon who ships with USPS.
These are basically the same. The Shopify emails contain the tracking number and status updates

**Comment by firstof9** [comment]
> I am not receiving a dedicated DHL/DPD/whatever mail, only a custom mail from the seller. Is there any way to deal with that?

I know for shippers like DHL you sign up to get notices from them when something is en-route to your address, similar to UPS.


> Neither is a seller on Amazon who ships with USPS.

While true, Amazon does have their own delivery drivers as well,  in the US at least.

**Comment by jaredcat** [comment]
> While true, Amazon does have their own delivery drivers as well, in the US at least.

I'm not referring to Prime sales. Amazon sellers can also sell and ship from their home/business just like Shopify stores. The emails operate the same in this case.

**Comment by firstof9** [comment]
Correct, in which case you'd get a tracking email from USPS/UPS/FedEx (if you enabled the notifications for your shipping address).
